<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135, 255), stop:0.427447 rgba(41, 61, 132, 235), stop:1 rgba(155, 79, 165, 255));</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: white;
font-size: 19px;</string>
   </property>
   <property name="text">
    <string>Specify the new rule (leave blank if unneeded)</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>411</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 30);
border-radius: 10%;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_order">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 6px;
color: white;
font-size: 16pt;</string>
    </property>
    <property name="placeholderText">
     <string>Order number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_direction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 6px;
color: white;
font-size: 16pt;</string>
    </property>
    <property name="placeholderText">
     <string>Packet direction</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_protocol">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 6px;
color: white;
font-size: 16pt;</string>
    </property>
    <property name="placeholderText">
     <string>Protocol</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_src_ips">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 6px;
color: white;
font-size: 16pt;</string>
    </property>
    <property name="placeholderText">
     <string>Source ip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dst_ips">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 6px;
color: white;
font-size: 16pt;</string>
    </property>
    <property name="placeholderText">
     <string>Destination ip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_src_ports">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 6px;
color: white;
font-size: 16pt;</string>
    </property>
    <property name="placeholderText">
     <string>Source port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dst_ports">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 6px;
color: white;
font-size: 16pt;</string>
    </property>
    <property name="placeholderText">
     <string>Destination port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 6px;
color: white;
font-size: 16pt;</string>
    </property>
    <property name="placeholderText">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_annotation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 6px;
color: white;
font-size: 16pt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="placeholderText">
     <string>Annotation</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="addRuleBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 128);
color: white;
font-size: 26px;
border-radius: 10%;
}

QPushButton::pressed {
background-color: rgba(255, 255, 255, 172);
color: white;
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
